frontend/src/libs/hooks/hooks.ts

```diff
@@ -3,5 +3,5 @@ export { useAppForm } from './use-app-form/use-app-form.hook.js';
 export { useAppSelector } from './use-app-selector/use-app-selector.hook.js';
 export { useStepper } from './use-app-stepper/use-app-stepper.hook.js';
 export { useCallback, useEffect, useMemo, useRef, useState } from 'react';
-export { useController as useFormController } from 'react-hook-form';
+export { useForm, useController as useFormController } from 'react-hook-form';
```

Comment:
```suggestion
export { useController as useFormController } from 'react-hook-form';
```
use `useAppForm`

Reply:
`useAppForm` using validationSchema, I need to add validation schema on search?

Reply:
better. you can make this param as optional